### PR TITLE
Install docs spruce-up & VS for Mac and VSCode mentions

### PR DESCRIPTION
**Pull request by donnie-msft**
Fixes https://github.com/NuGet/docs.microsoft.com-nuget/issues/1841

Spruced up the document overall, eliminated some redundancy. VSMac and VSCode are now line-items in the table, rather than just mentioned in a paragraph.

VS2010 - took the time to install it so here's a screenshot for reference:
image

### CODE REVIEW

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[5d2fd76](https://github.com/NuGet/docs.microsoft.com-nuget/commits/5d2fd76879a56b8dc78a01dfdd9fd44b571a5ee8)_: 

:clock10: **Pending**: waiting for processors ([11](https://ops.microsoft.com/#/statistic) builds ahead of you) 

 :warning: **Docs Build is busy**, currently there are 11 builds ahead of this one, for more information you can view the Build queue graph on the [Docs Portal](https://ops.microsoft.com/#/statistic).

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[5d2fd76](https://github.com/NuGet/docs.microsoft.com-nuget/commits/5d2fd76879a56b8dc78a01dfdd9fd44b571a5ee8)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/install-nuget-client-tools.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-donnie-msft-vsForMacDocs/docs/install-nuget-client-tools.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/nuget/install-nuget-client-tools?branch=pr-en-us-1920) | [Details](#user-content-201c3593-d6ccec35)

<a id="201c3593-d6ccec35"></a>
### [docs/install-nuget-client-tools.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-donnie-msft-vsForMacDocs/docs/install-nuget-client-tools.md)
 - **Line 20**: **[Warning]** `````Cannot find bookmark '#visual-studio' in 'install-nuget-client-tools.md'`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C28%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202002281923326728-1920%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=F3pDND26w2fk1TMIaKoZwiv%2BPTVfsuzi%2FFl8u2yMNJ0%3D&st=2020-02-28T19%3A19%3A11Z&se=2020-03-30T19%3A24%3A11Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[ed53cae](https://github.com/NuGet/docs.microsoft.com-nuget/commits/ed53caee0580485ce39a9e280b248b4057cc38f6)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/install-nuget-client-tools.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-donnie-msft-vsForMacDocs/docs/install-nuget-client-tools.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/install-nuget-client-tools?branch=pr-en-us-1920) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C2%5C28%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202002281935136575-1920%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=TIEPxHqGdtAKBZihjfzdeqpgZc1OH5D%2FwTS5u%2BFpe5c%3D&st=2020-02-28T19%3A30%3A51Z&se=2020-03-30T19%3A35%3A51Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/install-nuget-client-tools.md [hunk 1]
```diff
@@ -17,11 +17,25 @@ To work with NuGet, as a package consumer or creator, you can use command-line i
...
+| [Visual Studio for Mac](/visualstudio/mac/nuget-walkthrough) | On Mac, certain NuGet capabilities are built-in directly. Package Manager Console is not presently available. For other capabilities, use the `dotnet.exe` or `nuget.exe` CLI tools.  | [Visual Studio for Mac](https://visualstudio.microsoft.com/vs/mac/) |
+| [Visual Studio Code](https://code.visualstudio.com/docs) | On Windows, Mac, or Linux, NuGet capabilities are available through marketplace extensions, or use the `dotnet.exe` or `nuget.exe` CLI tools. | [Visual Studio Code](https://code.visualstudio.com/Download/)|
 
 The [MSBuild CLI](reference/msbuild-targets.md) also provides the ability to restore and create packages, which is primarily useful on build servers. MSBuild is not a general-purpose tool for working with NuGet.
 
+Package Manager Console commands work only within Visual Studio on Windows and do not work within other PowerShell environments.
```
Comment (nkolev92) [inline docs/install-nuget-client-tools.md]: Remove `commands` maybe?

```suggestion
Package Manager Console works only within Visual Studio on Windows and does not work within other PowerShell environments.
```

File: docs/install-nuget-client-tools.md [hunk 2]
```diff
@@ -17,11 +17,25 @@ To work with NuGet, as a package consumer or creator, you can use command-line i
...
+### Install on Visual Studio 2017 and newer
+Starting in Visual Studio 2017, the installer includes the NuGet Package Manager with any workload that employs .NET. To install separately, or to verify that the Package Manager is installed, run the Visual Studio installer and check the option under **Individual Components > Code tools > NuGet package manager**.
+
+### Install on Visual Studio 2015 and older
+NuGet Extensions for Visual Studio 2013 and 2015 can be downloaded from [https://dist.nuget.org/index.html](https://dist.nuget.org/index.html).
+
+For Visual Studio 2010 and earlier, install the "NuGet Package Manager for Visual Studio" extension. Note, if you can't see the extension in the first page of search results, try changing the Sort By dropdown to "Most Downloads", or an alphabetical sort.
+
 ## CLI tools
+A CLI tool is required to support NuGet features in the IDE. You can use either the `dotnet` CLI or the the `nuget.exe` CLI. The `dotnet` CLI is installed with some Visual Studio workloads, such as .NET Core. The `nuget.exe` CLI must be installed separately as described earlier.
```
Comment (nkolev92) [inline docs/install-nuget-client-tools.md]: > A CLI tool is required to support NuGet features in the IDE.

I don't really follow this...seems counter intuitive.

Comment (donnie-msft) [reply docs/install-nuget-client-tools.md]: I didn't write it, I just moved it. Can you suggest a rewrite?

File: docs/install-nuget-client-tools.md (hunk 1 above)
Comment (donnie-msft) [reply docs/install-nuget-client-tools.md]: Doesn't make sense. The point is that the commands are different. Can you double-check this?

File: docs/install-nuget-client-tools.md (hunk 2 above)
Comment (donnie-msft) [reply docs/install-nuget-client-tools.md]: Perhaps:
`To support some NuGet features, a CLI tool is required outside the IDE.`

?

File: docs/install-nuget-client-tools.md [hunk 3]
```diff
@@ -17,11 +17,25 @@ To work with NuGet, as a package consumer or creator, you can use command-line i
...
+## Visual Studio
+### Install on Visual Studio 2017 and newer
+Starting in Visual Studio 2017, the installer includes the NuGet Package Manager with any workload that employs .NET. To install separately, or to verify that the Package Manager is installed, run the Visual Studio installer and check the option under **Individual Components > Code tools > NuGet package manager**.
+
+### Install on Visual Studio 2015 and older
+NuGet Extensions for Visual Studio 2013 and 2015 can be downloaded from [https://dist.nuget.org/index.html](https://dist.nuget.org/index.html).
+
+For Visual Studio 2010 and earlier, install the "NuGet Package Manager for Visual Studio" extension. Note, if you can't see the extension in the first page of search results, try changing the Sort By dropdown to "Most Downloads", or an alphabetical sort.
+
 ## CLI tools
```
Comment (kartheekp-ms) [inline docs/install-nuget-client-tools.md]: `### CLI tools`

File: docs/install-nuget-client-tools.md (hunk 2 above)
Comment (kartheekp-ms) [reply docs/install-nuget-client-tools.md]: may be start the content with - "You can use either the `dotnet` CLI or the the `nuget.exe` CLI to support NuGet features in the IDE."

File: docs/install-nuget-client-tools.md (hunk 1 above)
Comment (zkat) [reply docs/install-nuget-client-tools.md]: I think the original looks good, tbh

File: docs/install-nuget-client-tools.md (hunk 3 above)
Comment (zkat) [reply docs/install-nuget-client-tools.md]: I believe this is actually the correct level for this header.

File: docs/install-nuget-client-tools.md (hunk 2 above)
Comment (karann-msft) [inline docs/install-nuget-client-tools.md]:
```suggestion
You can use either the `dotnet` CLI or the `nuget.exe` CLI to support NuGet features in the IDE. The `dotnet` CLI is installed with some Visual Studio workloads, such as .NET Core. The `nuget.exe` CLI must be installed separately as described earlier.
```

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[6767511](https://github.com/NuGet/docs.microsoft.com-nuget/commits/6767511f9ab07c480eaa6ac1717e53a7cc93d851)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/install-nuget-client-tools.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-donnie-msft-vsForMacDocs/docs/install-nuget-client-tools.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/install-nuget-client-tools?branch=pr-en-us-1920) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C3%5C2%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202003021810434209-1920%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=yHGDvcxN0C%2BbzB%2Bhh4Lp9bMeFHkRzlhgjdgyBjqxs4U%3D&st=2020-03-02T18%3A06%3A21Z&se=2020-04-02T18%3A11%3A21Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**